contrib/repairStatus.py

```diff
@@ -0,0 +1,199 @@
+from __future__ import print_function
+import json
+import argparse
+import requests
+import re
+import logging as log
+from datetime import datetime, timedelta
+
+
+def generate_range_repair_query (node, start_time, current_time):
+  query_json = {
+    "query": {
+        "filtered": {
+            "query": {
+                "bool": {
+                    "should": [
+                        {
+                            "query_string": {
+                                "query": 'level:"info" AND (*)'
+                            }
+                        }
+                    ]
+                }
+            },
+            "filter": {
+                "bool": {
+                    "must": [
+                        {
+                            "range": {
+                                "@timestamp": {
+                                    "from": start_time,
+                                    "to": current_time
+                                }
+                            }
+                        },
+                        {
+                            "fquery": {
+                                "query": {
+                                    "query_string": {
+                                        "query": 'host:("' + node + '")'
+                                    }
+                                },
+                                "_cache": True
+                            }
+                        },
+                        {
+                            "fquery": {
+                                "query": {
+                                    "query_string": {
+                                        "query": 'method:("syncComplete")'
+                                    }
+                                },
+                                "_cache": True
+                            }
+                        },
+                        {
+                            "fquery": {
+                                "query": {
+                                    "query_string": {
+                                        "query": 'message:("is fully synced")'
+                                    }
+                                },
+                                "_cache": True
+                            }
+                        }
+                    ]
+                }
+            }
+        }
+    }
+  }
+
+  return json.dumps(query_json)
+
+
+# Execute an ES _count to return the number of ranges that have finished repairing
```

Comment:
Rather than a comment before the function, the general convention is to put a docstring in the function itself, like

```python
def calculate_range_repairs(host, port, node, start_time, current_time):
    """
    Execute an ES _count to return the number of ranges that have finished repairing
    """
```